packages/app-store/locations.ts

```diff
@@ -303,13 +303,13 @@ export const getHumanReadableLocationValue = (
 
   // Just in case linkValue is a `locationType.type`(for old bookings)
   const eventLocationType = getEventLocationType(linkValue);
-
+  const isDefault = eventLocationType?.default;
   if (eventLocationType) {
     // If we can find a video location based on linkValue then it means that the linkValue is something like integrations:google-meet and in that case we don't have the meeting URL to show.
     // Show a generic message in that case.
-    return `${eventLocationType.label}`;
+    return isDefault ? translationFunction(eventLocationType.label) : `${eventLocationType.label}`;
```

Comment:
@anikdhabal I just thought after the merge. We should translate both the labels, default and non-default. Right now  App location labels aren't using translation keys but when they start to do it, they will face the same issue.
<img width="566" alt="Screenshot 2023-08-01 at 5 00 49 PM" src="https://github.com/calcom/cal.com/assets/1780212/845edf85-7aac-4268-a58d-d2d3c2586de4">

Reply:
> @anikdhabal I just thought after the merge. We should translate both the labels, default and non-default. Right now App location labels aren't using translation keys but when they start to do it, they will face the same issue. <img alt="Screenshot 2023-08-01 at 5 00 49 PM" width="566" src="https://user-images.githubusercontent.com/1780212/257501011-845edf85-7aac-4268-a58d-d2d3c2586de4.png">

Yes, it's a valid point. @hariombalhara okay will make a PR then.